### ISSUE TITLE
about whole dataset training

### ISSUE THREAD

**Issue by Kciiiman** [NUS-HPC-AI-Lab/DATM#16]
Hello Sir. Is there a general framework for training the wholedataset about CIFAR and Tiny-ImageNet? It seems a bit different when I train it using SGD. If you can tell me roughly about it, that would be really great.

**Comment by GzyAftermath** [comment]
Hi, could you specify what differences you're referring to?

By default in our script, the model uses GSAM during the buffer stage of training, while SGD is used for evaluation training. I assume the differences you mentioned might be due to the use of ZCA whitening and DSA augmentation. Additionally, some hyperparameters could also affect the results.

**Comment by Kciiiman** [comment]
Excuse me, sir. May be there is a problem in my presentation, I mean I train full dataset about CIFAR and Tiny-ImageNet, I used SGD and Learning Rate Scheduler but my result is not 84, 56, 37. what should I configure to achieve the result of full dataset in your paper? Thanks for your answer.

**Comment by GzyAftermath** [comment]
> Excuse me, sir. May be there is a problem in my presentation, I mean I train full dataset about CIFAR and Tiny-ImageNet, I used SGD and Learning Rate Scheduler but my result is not 84, 56, 37. what should I configure to achieve the result of full dataset in your paper? Thanks for your answer.

Our baseline results are obtained from previous work [1][2], based on my experiments I remembered the hyper-parameters don't influence the baseline results too much (on cifar-10 the results are always between 83-85). If you want to reproduce the exact same results, maybe you can refer to the settings in [1] and [2].

[1] Dataset Distillation by Matching Training Trajectories.
[2] Dataset Condensation with Gradient Matching.
